index.html

```diff
@@ -333,8 +325,38 @@
 				</section>
 
 				<section>
-					<h1>Feedback welcome!</h1>
-					<h2><a href="https://github.com/hbz/swib16-workshop/issues/new">https://github.com/hbz/swib16-workshop/issues/new</a></h2>
+					<section>
+						<h1>Improving openness of our own projects</h1>
+					</section>
+					<section>
+						<h1>License</h1>
+						<p>TODO</p>
+					</section>
+					<section>
+						<h1>Contributing</h1>
+						<p>TODO</p>
+					</section>
+					<section>
+						<h1>Issue templates</h1>
+						<p>TODO</p>
+					</section>
+					<section>
```

Comment:
Creating the issue template I found https://help.github.com/articles/helping-people-contribute-to-your-project/. This also lists PR template and a code of conduct. We should think about also adding these here.